layouts/partials/footer.html

```diff
@@ -3,7 +3,7 @@
 				<div class="row">
 					<div class="col-xs-12 text-xs-center">
 						<p>
-							Copyright &copy; {{ .Now.Year }} <a href="{{ .Site.Params.website }}">{{ .Site.Params.author }}</a>. All rights reserved. Made with <i class="fa fa-heart" aria-hidden="true"></i> and <a href="https://gohugo.io">Hugo</a>. Sponsored by <a href="http://inblockchain.com">INBlockchain</a>.
+							Copyright &copy; {{ now.Year }} <a href="{{ .Site.Params.website }}">{{ .Site.Params.author }}</a>. All rights reserved. Made with <i class="fa fa-heart" aria-hidden="true"></i> and <a href="https://gohugo.io">Hugo</a>. Sponsored by <a href="http://inblockchain.com">INBlockchain</a>.
```

Comment:
fix `ERROR 2017/08/02 07:25:20 Page's Now is deprecated and will be removed in Hugo 0.26. Use now (the template func).` error from hugo.